### PR TITLE
Spec and PY update

### PR DESCRIPTION
**Pull request by JellyStanchina**
Bork, Please take a look at the latest files.
I think I have given what is currently requested.
Please scrutinize.

Thanks,
Joey

### CODE REVIEW

File: Python/RSSFeedReader/RSSFeedReader.py [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
-# Request RSS Feed for set URL and collect output
-
-# print response output
+# Set the RSS FeedURL
+URL = 'http://www.motortrend.com/widgetrss/motortrend-stories.xml'
 
+# Request RSS Feed for set URL and collect output
+try:
+  response = requests.get(URL)
+  response.raise_for_status()
```
Comment (borkgit) [inline Python/RSSFeedReader/RSSFeedReader.py]: What does this line do?

Comment (borkgit) [reply Python/RSSFeedReader/RSSFeedReader.py]: I am talking about line 12

**Review by borkgit — COMMENTED** [review]
Added a question

**Review by borkgit — COMMENTED** [review]
Does the etree object have ways of querying the xml data?

File: Python/RSSFeedReader/RSSFeedReader.py (hunk 1 above)
Comment (JellyStanchina) [reply Python/RSSFeedReader/RSSFeedReader.py]: I found documentation here: https://2.python-requests.org/en/master/user/quickstart/
- No MSDN type page yet ;-)

It is used to raise the http_error.
I tried to cause and use the "raise" output and nothing seemed to work.
I did however add error handling using .status_code

Comment (borkgit) [reply Python/RSSFeedReader/RSSFeedReader.py]: Here is the msdn page...

https://docs.python.org/3/library/index.html

Comment (borkgit) [reply Python/RSSFeedReader/RSSFeedReader.py]: Looks like "requests" is a third party library.  It looks like there are core objects that you can use for an http request. I would try using those instead to start out with.

Comment (borkgit) [reply Python/RSSFeedReader/RSSFeedReader.py]: check this page...

https://docs.python.org/3/library/urllib.request.html#module-urllib.request

Comment (JellyStanchina) [reply Python/RSSFeedReader/RSSFeedReader.py]: That would explain why I could not find requests on this page ;-)
I will re-code with this native urllib.request

**Comment by JellyStanchina** [comment]
Still to resolve for borkgit:
Does the etree object have ways of querying the xml data?

File: Python/RSSFeedReader/RSSFeedReader.py [hunk 2]
```diff
@@ -1,9 +1,36 @@
...
 
-# print response output
+# Request RSS Feed for set URL and collect output
+try:
+  response = urllib.request.urlopen(URL)
+  status_code = response.getcode()
+  if not status_code == 200:
+    print(f'URL Request Open Error: {status_code}')
+    exit()
+  with response as RSSPage:
```
Comment (borkgit) [inline Python/RSSFeedReader/RSSFeedReader.py]: Explain the with statement above.

Comment (JellyStanchina) [reply Python/RSSFeedReader/RSSFeedReader.py]: Turns out this is not needed at all.
I have removed it.

**Comment by JellyStanchina** [comment]
Recursion replaced with use of .findall()

**Review by borkgit — APPROVED** [review]

**Pull request merged**